php_phongo.c

```diff
@@ -1276,17 +1281,7 @@ void php_phongo_cursor_to_zval(zval *retval, php_phongo_cursor_t *cursor) /* {{{
 		add_assoc_null_ex(retval, ZEND_STRS("cursor"));
 	}
 
-	if (cursor->firstBatch) {
-		php_phongo_bson_state  state = PHONGO_BSON_STATE_INITIALIZER;
-
-		MAKE_STD_ZVAL(state.zchild);
-		bson_to_zval(bson_get_data(cursor->firstBatch), cursor->firstBatch->len, &state);
-		add_assoc_zval_ex(retval, ZEND_STRS("firstBatch"), state.zchild);
-	} else {
-		add_assoc_null_ex(retval, ZEND_STRS("firstBatch"));
-	}
 	add_assoc_long_ex(retval, ZEND_STRS("server_id"), cursor->server_id);
-	add_assoc_bool_ex(retval, ZEND_STRS("is_command_cursor"), cursor->is_command_cursor);
```

Comment:
@derickr: Take note that I'm removing "firstBatch" and "is_command_cursor" from the debug output. If this is merged, I will create HHVM tickets for you.